### DISCUSSION TITLE
採購班題庫可以再更新一下囉

### DISCUSSION THREAD

**Discussion by ans00047** [sdwh/sdwh.github.io#34] [General]
感謝您做了這個題庫系統以供練習！
今天上課時老師查詢3月到6月之間新增的考題，已經新增了80幾題。

**Comment by sdwh** [comment]
哈囉，更新好囉。

重新整理「採購專業人員基礎證照 | 測驗練習」看到 **(112 年 6月 最新版) 的標題**後，按下**「更新題庫」**就可以囉😀
